mabl snippets/randomNumberFromRange.js

```diff
@@ -0,0 +1,9 @@
+/**
+* Generate random number between a range
```

Comment:
We can do this with [string templates in mabl](https://help.mabl.com/docs/valid-expressions#:~:text=Example%3A%20%7B%7BrandomInt(0%2C%2011)%7D%7D%20will%20return%20a%20random%20integer%20between%200%2D10.)

We can keep this, but I would link to that to show there is a better way.